helm/values.yaml

```diff
@@ -0,0 +1,45 @@
+replicaCount: 1
+
+image:
+  repository: reloader
+  pullPolicy: Never
+  tag: "latest"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  create: true
+  annotations: {}
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: NodePort
```

Comment:
This value must be ClusterIp: https://github.com/pagopa/selfcare-ms-product/blob/111c2a7d3516ec581747fead5918727b4b0bc438/manifests/service.yml#L6

Reply:
Yep, do you think we should remove the service at all? I mean, could the service be considered as part of the infrastructure and not of the microservice?

Reply:
No, the service is always part of the project. The ingress can be something shared